test.sh

```diff
@@ -229,23 +254,10 @@ function nightly_sync_perf_tests() {
     local npthreads=8
     local tree_size=8 # GiB
     test_descr="tree-size ${tree_size} GiB, ${npthreads} pthreads"
-=======
-    run_with_timing "Performance (sync) test ${test_descr}" \
-            $BINDIR/driver_test splinter_test --perf \
-                                          --max-async-inflight 0 \
-                                          --num-insert-threads ${nins_t} \
-                                          --num-lookup-threads ${nlookup_t} \
-                                          --num-range-lookup-threads ${nrange_lookup_t} \
-                                          --lookup-positive-percent 10 \
-                                          --db-capacity-gib 60 \
-                                          --db-location ${dbname}
-    rm ${dbname}
-
->>>>>>> respond to all comments and feature requests
```

Comment:
Merge issue ... Deleted messed-up code.

(NOTE: I am still wading thru diffs, and merges, trying to stabilize CI-runs. So this commit is not quite ready for a review.

Ideally, once I have got all CI runs coming out clean, @rtjohnso would want to scrutinize this commit's diffs, and cherry-pick it into his branch, and then push it forward. Until then, this work is really under dev.)